web-marketplace/src/components/organisms/CreditBatchesSection/CreditBatchesSection.types.ts

```diff
@@ -9,7 +9,7 @@ import {
 import { UseStateSetter } from 'types/react/use-state';
 
 export type ProjectMiddleSectionProps = {
-  offChainProject?: Maybe<ProjectFieldsFragment>;
+  offChainProject?: Maybe<Partial<ProjectFieldsFragment>>;
```

Comment:
instead of adding Partial here, I think we should probably standardize the project queries to use the same fragment, in particular ProjectById because otherwise, we might miss some info on the uuid based project pages (like documents for instance, which are not part of the current `ProjectById` query)

Reply:
yeah I think we should do that, this is not only a refactor but making sure all the elements are queried and displayed accordingly on the project page

Reply:
I believe you would just need to use our existing projectFields fragment in the ProjectById query, leaving this part unchanged